make.sh

```diff
@@ -117,6 +117,9 @@ rename_folder "${CONSTANT_PROJECT_NAME}" "${PROJECT_NAME_NO_SPACES}"
 mkdir -p "${PROJECT_NAME_NO_SPACES}Tests/Sources/Mocks/Sourcery"
 touch "${PROJECT_NAME_NO_SPACES}Tests/Sources/Mocks/Sourcery/AutoMockable.generated.swift"
 
+# Add R.generated.swift file
+touch "${PROJECT_NAME_NO_SPACES}/Sources/Supports/Helpers/Rswift/R.generated.swift"
```

Comment:
We should create the folder `Rswift` first right?

```suggestion
mkdir -p "${PROJECT_NAME_NO_SPACES}/Sources/Supports/Helpers/Rswift"
touch "${PROJECT_NAME_NO_SPACES}/Sources/Supports/Helpers/Rswift/R.generated.swift"
```

Reply:
@suho it's already included in our template, so just need touch the file only
![Screen Shot 2021-11-24 at 14 19 31](https://user-images.githubusercontent.com/17875522/143192646-32ff67be-2e7d-48bb-8ccb-18d8b8806cd9.png)
te